### PR TITLE
98439-deprecate-openzaak-module

### PR DESCRIPTION
**Pull request by theo-ritense**

### CODE REVIEW

File: getting-started/modules/zgw/openzaak.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 In order to use OpenZaak, the OpenZaak module needs to be added as a dependency.
 The following can be added to your project, depending on whether Maven or Gradle is used:
 
+> **_NOTE_**: The OpenZaak module has been deprecated.
```
Comment (marijnritense) [inline getting-started/modules/zgw/openzaak.md:8]:
```suggestion
> **_NOTE:_** The OpenZaak module has been deprecated.
```

File: introduction/modules/modules.md [hunk 2]
```diff
@@ -277,11 +277,16 @@ API standard for storage of and access to object types, without having to create
 provided that allows for connecting to applications that provide an Objecttypen API.
 
 ### OpenZaak
+> **_NOTE_**: This module has been deprecated
```
Comment (marijnritense) [inline introduction/modules/modules.md:280]:
```suggestion
> **_NOTE:_** This module has been deprecated
```

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ The following features were added:
   The assignees of a user task are now saved in the database by their user ID. When implementations use keycloak, the
   assignee column from the task table in the databases is automatically migrated from email to user ID.
 
+* **New endpoint to retrieve Zaaktypen**
+  A new endpoint has been added to the zaken-api module to retrieve zaaktypen: `/api/management/v1/zgw/zaaktype`. 
+  This replaces the `/api/v1/openzaak/zaaktype` endpoint from the openzaak module, which is now deprecated.
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-backend-libraries.md:23]: Can you also mention this under deprecations?

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md [hunk 4]
```diff
@@ -47,6 +55,11 @@ The following breaking changes were introduced:
   `valtimo-dependencies` and `valtimo-gzac-depenencies` should no longer be used for `dependencyManagement` or `platform`. 
   This functionality has been replaced by [valtimo-dependency-versions](../../../getting-started/modules/core/valtimo-dependency-versions.md).
 
+* **Moved KvKProvider and BsnProvider.**
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-backend-libraries.md:58]:
```suggestion
* **Moved KvKProvider and BsnProvider**
```

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md [hunk 5]
```diff
@@ -47,6 +55,11 @@ The following breaking changes were introduced:
...
+  The `KvKProvider` and `BsnProvider` and implementations (`ZaakKvKProvider` and `ZaakBsnProvider`)have been moved to the `zaken-api` module. 
+  The [objects-api module](../../../getting-started/modules/zgw/objects-api.md) has been changed to use the relocated interfaces.
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-backend-libraries.md:60]:
```suggestion
  The [objects-api module](/getting-started/modules/zgw/objects-api.md) has been changed to use the relocated interfaces.
```

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md [hunk 6]
```diff
@@ -47,6 +55,11 @@ The following breaking changes were introduced:
...
+  Please be aware that the new implementations require at least one Zaken API Plugin to be configured.
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-backend-libraries.md:61]: Maybe make this a migration note?

File: getting-started/modules/zgw/openzaak-plugin-authentication.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# Objecten API Authentication
```
Comment (Klaas-Ritense) [inline getting-started/modules/zgw/openzaak-plugin-authentication.md:1]:
```suggestion
# Open Zaak API Authentication
```

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md (hunk 3 above)
Comment (theo-ritense) [reply release-notes/major12/12.0.0/valtimo-backend-libraries.md:55]: added

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md (hunk 6 above)
Comment (theo-ritense) [reply release-notes/major12/12.0.0/valtimo-backend-libraries.md:93]: added

File: getting-started/modules/zgw/openzaak-plugin-authentication.md (hunk 7 above)
Comment (theo-ritense) [reply getting-started/modules/zgw/openzaak-plugin-authentication.md]: changed

File: release-notes/major12/12.0.0/valtimo-frontend-libraries.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Frontend libraries 12.0.0
```
Comment (Klaas-Ritense) [inline release-notes/major12/12.0.0/valtimo-frontend-libraries.md:1]: Merge conflict?

Comment (theo-ritense) [reply release-notes/major12/12.0.0/valtimo-frontend-libraries.md]: fixed.

**Review by marijnritense — APPROVED** [review]

**Pull request merged**